heliostats-requirements.txt

```diff
@@ -39,14 +39,14 @@ gunicorn==19.3.0
 ipdb==0.8.1
 ipython==3.2.1
 jmespath==0.9.4
+mod_wsgi==4.7.1
 nose==1.3.7
 nose-exclude==0.5.0
 numpy==1.17.4
 openpyxl==2.5.0
 pandas==0.18.1
 pathspec==0.3.3
-psycopg2==2.7.4
-psycopg2-binary==2.7.4
+psycopg2==2.8.3
```

Comment:
Curious why we removed psycopg2-binary here. I thought this was the preferred way to install?

Reply:
I think, normally, yes. But psycopg-binary causes Apache to break.
The legacy `heliostats` contained a workaround for this - they installed psycopg/psycopg-binary==2.7.4 (from requirements files), uninstalled psycopg-binary, then reinstalled psycopg 2.8.3.

The docker container is skipping those steps and just installed psycopg 2.8.3 straightaway

legacy code:
```yaml
- name: Heliostats | Install heliostats pip requirements.
  pip:
    virtualenv: "{{ heliostats_root }}/venv"
    virtualenv_python: python3.5
    requirements: "{{ heliostats_root }}/{{ pip_requirements_filename }}"
  environment:
    TMPDIR: /data

- name: Heliostats | Install marvin pip requirements.
  pip:
    virtualenv: "{{ heliostats_root }}/venv"
    virtualenv_python: python3.5
    requirements: "{{ heliostats_root }}/marvin/{{ marvin_pip_requirements_filename }}"
  environment:
    TMPDIR: /data

- name: "Heliostats | Fix psycopg version (TODO: fix inside requirements.txt)"
  pip:
    virtualenv: "{{ heliostats_root }}/venv"
    virtualenv_python: python3.5
    name: psycopg2-binary
    state: absent
  environment:
    TMPDIR: /data
  tags: psycopg-fix

- name: "Heliostats | Fix psycopg version (TODO: fix inside requirements.txt)"
  pip:
    virtualenv: "{{ heliostats_root }}/venv"
    virtualenv_python: python3.5
    name: psycopg2
    version: 2.8.3
  environment:
    TMPDIR: /data
  tags: psycopg-fix

```

Reply:
> Curious why we removed psycopg2-binary here. I thought this was the preferred way to install?

I think they stopped recommending psycopg2-binary starting with version 2.8.3, which is what we need to avoid mod_wsgi crashes.